core/src/main/java/org/apache/iceberg/MetricsConfig.java

```diff
@@ -46,6 +47,35 @@ public static MetricsConfig getDefault() {
     return spec;
   }
 
+  public static Map<String, String> updateProperties(Map<String, String> props, List<String> deletedColumns,
+                                                     Map<String, String> renamedColumns) {
+    if (props.keySet().stream().noneMatch(key -> key.startsWith(TableProperties.METRICS_MODE_COLUMN_CONF_PREFIX))) {
+      return props;
+    } else {
+      Map<String, String> updatedProperties = Maps.newHashMap();
+      // Put all of the non metrics columns we aren't modifying
+      props.keySet().forEach(key -> {
+        if (key.startsWith(TableProperties.METRICS_MODE_COLUMN_CONF_PREFIX)) {
+          String columnAlias = key.replaceFirst(TableProperties.METRICS_MODE_COLUMN_CONF_PREFIX, "");
+          if (renamedColumns.get(columnAlias) != null) {
+            // The name has changed.
+            String newKey = TableProperties.METRICS_MODE_COLUMN_CONF_PREFIX +
```

Comment:
Can we add a static import for `METRICS_MODE_COLUMN_CONF_PREFIX` like we have for other two props in this class? That way, the class will be a little bit more consistent and this logic will fit on one line.